Formula/node_exporter.rb

```diff
@@ -0,0 +1,64 @@
+class NodeExporter < Formula
+  desc "Prometheus exporter for machine metrics"
+  homepage "https://prometheus.io/"
+  url "https://github.com/prometheus/node_exporter/archive/v0.16.0.tar.gz"
+  sha256 "2ed1c1c199e047b1524b49a6662d5969936e81520d6613b8b68cc3effda450cf"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/prometheus").mkpath
+    ln_sf buildpath, buildpath/"src/github.com/prometheus/node_exporter"
+
+    system "go", "build", "github.com/prometheus/node_exporter"
+    bin.install "node_exporter"
+  end
+
+  plist_options :startup => "sudo brew services start node_exporter"
```

Comment:
does it really fail without `sudo`? It would probably be nicer to use `plist_options :manual` with and not to use `sudo -u nobody` in the plist

Reply:
When started as a user agent, `sudo` fails (AFAIK; it's been a while since I wrote the code).

I've changed it to `:manual` and without the `sudo` -parts.